### PR TITLE
Extended chargepoint and new dev_sys service.

### PR DESCRIPTION
**Pull request by boguslaw-wojcik**

### CODE REVIEW

File: adapter/service/chargepoint/model.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	SessionEnergy         float64
+	PreviousSessionEnergy float64
+	StartedAt             time.Time
+	FinishedAt            time.Time
+	OfferedCurrent        int64
+}
+
+// reportProperties returns a map of report properties taking into consideration capabilities of the chargepoint.
+func (r *SessionReport) reportProperties(supportsAdjustingCurrent bool) map[string]string {
+	properties := make(map[string]string)
```
Comment (rstraczynski) [inline adapter/service/chargepoint/model.go:60]: SessionEnergy shouldn't be included here, right?

File: adapter/service/chargepoint/routing.go [hunk 2]
```diff
@@ -214,6 +217,44 @@ func handleCmdCableLockGetReport(serviceRegistry adapter.ServiceRegistry) router
...
+// handleCmdCurrentSessionSetCurrent returns a handler responsible for handling the command.
+func handleCmdCurrentSessionSetCurrent(serviceRegistry adapter.ServiceRegistry) router.MessageHandler {
+	return router.NewMessageHandler(
+		router.MessageProcessorFn(func(message *fimpgo.Message) (*fimpgo.FimpMessage, error) {
+			chargepoint, err := getService(serviceRegistry, message)
+			if err != nil {
+				return nil, err
+			}
+
+			current, err := message.Payload.GetIntValue()
```
Comment (rstraczynski) [inline adapter/service/chargepoint/routing.go:238]: Is current going to be always an integer?

File: adapter/service/devsys/routing_test.go [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+	"github.com/futurehomeno/cliffhanger/adapter/service/devsys"
+	"github.com/futurehomeno/cliffhanger/router"
+	"github.com/futurehomeno/cliffhanger/task"
+	adapterhelper "github.com/futurehomeno/cliffhanger/test/helper/adapter"
+	mockedadapter "github.com/futurehomeno/cliffhanger/test/mocks/adapter"
+	mockeddevsys "github.com/futurehomeno/cliffhanger/test/mocks/adapter/service/devsys"
+	"github.com/futurehomeno/cliffhanger/test/suite"
+)
+
+func TestRouteCarCharger(t *testing.T) { //nolint:paralleltest
```
Comment (rstraczynski) [inline adapter/service/devsys/routing_test.go:20]: Please fix naming here.

**Review by rstraczynski — CHANGES_REQUESTED** [review]
Two questions and one change.

File: adapter/service/chargepoint/model.go (hunk 1 above)
Comment (boguslaw-wojcik) [reply adapter/service/chargepoint/model.go:60]: Session Energy is reported as main value. This is helper for optional properties.

File: adapter/service/chargepoint/routing.go (hunk 2 above)
Comment (boguslaw-wojcik) [reply adapter/service/chargepoint/routing.go:238]: Yes.

**Review by rstraczynski — APPROVED** [review]

**Pull request merged**